packages/moonstone/VirtualList/virtualListItemsRenderer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+/* eslint-disable enact/prop-types */
+const listItemsRenderer = (props) => {
+	const {
+		cc,
+		handlePlaceholderFocus,
+		itemContainerRef: initUiItemContainerRef,
+		needsScrollingPlaceholder,
+		primary,
+		role,
+		SpotlightPlaceholder
+	} = props;
+
+	return (
+		<React.Fragment>
```

Comment:
We can use `<>` here.
https://en.reactjs.org/docs/fragments.html#short-syntax

Reply:
Good point! I fixed it.